### PR TITLE
fix: erase unused functions

### PR DESCRIPTION
**Pull request by rokamu623**
## Description
The following ticket changed the implementation of Histogram and ResponseTime, but the code before the change remained, so it was removed.
- https://tier4.atlassian.net/browse/RT2-889
- https://tier4.atlassian.net/browse/RT2-824
- https://tier4.atlassian.net/browse/RT2-223

image

## Pre-review checklist for the PR author

- [x] I've confirmed the [contribution guidelines](https://github.com/tier4/caret/blob/main/.github/CONTRIBUTING.md).
- [x] The PR is ready for review.

## In-review checklist for the PR reviewers

The PR reviewers **must** check the checkboxes below before approval.

- [x] The PR has been properly tested.
- [x] The PR has been reviewed.

## Post-review checklist for the PR author

The PR author **must** check the checkboxes below before merging.

- [x] (Optional) The PR has been properly tested with CARET_report verification.
- [ ] There are no open discussions or they are tracked via tickets.
- [x] The PR is ready for merge.

After all checkboxes are checked, anyone who has write access can merge the PR.

### CODE REVIEW

File: src/caret_analyze/record/records_service/response_time.py [hunk 1]
```diff
@@ -1014,242 +911,3 @@ def _create_worst_to_best_case_record(
...
-        t_in = np.array(t_, dtype=np.int64)
-
-        t_out_ = records.get_column_series(output_column)
-        t_out = np.array(t_out_, dtype=np.int64)
-
-        latency = t_out - t_in
-
-        return t_in, latency
-
-    def to_best_case_records(self) -> RecordsInterface:
```
Comment (taro-yu) [inline src/caret_analyze/record/records_service/response_time.py:1052]: In the ResponseTime class, shouldn't the following section be deleted as it seems to use the erased to_best_case_records?
` def to_best_case_records(self) -> RecordsInterface:`

File: src/caret_analyze/record/records_service/response_time.py [hunk 2]
```diff
@@ -509,8 +506,6 @@ def __init__(
         response_map = ResponseMap(records, columns)
         self._response_map_all = ResponseMapAll(records, columns)
         self._records = ResponseRecords(response_map)
-        self._timeseries = ResponseTimeseries(self._records)
```
Comment (taro-yu) [inline src/caret_analyze/record/records_service/response_time.py:512]: In the following two functions in the ResponseTime class, this deleted part is used:
- `to_best_case_records()`
- `to_worst_case_records`

**Review by taro-yu — APPROVED** [review]
LGTM!

**Review by xygyo77 — APPROVED** [review]
image
Graph drawing was also checked.

**Pull request merged**